windows/.gitignore

```
@@ -10,6 +10,7 @@
 [Dd]ebugPublic/
 [Rr]elease/
 [Rr]eleases/
+**/AppPackages/**
```

Do we really need all the asterisks? Doesn't the following already catch all?
```suggestion
AppPackages/
```